src/inc/volatile.h

```diff
@@ -120,7 +120,7 @@ T VolatileLoad(T const * pt)
     STATIC_CONTRACT_SUPPORTS_DAC_HOST_ONLY;
 
 #ifndef DACCESS_COMPILE
-#if defined(_ARM64_) && defined(__GNUC__)
+#if defined(_ARM64_) && defined(__GNUC__) && !(defined(__BIONIC__))
```

Comment:
@janvorli Thanks for the info.  Is this a bug or something specific to Android?

@jkotas Okay to leave in the `__BIONIC__` and `__ANDROID__` compile options in toolchain.cmake in case they're needed in the future? They're standard compile options when cross building.